src/dmd/expressionsem.d

```diff
@@ -12152,14 +12152,15 @@ private bool checkAddressVar(Scope* sc, UnaExp exp, VarDeclaration v)
                 // Taking the address of v means it cannot be set to 'scope' later
                 v.storage_class &= ~STC.maybescope;
                 v.doNotInferScope = true;
-                if (exp.e1.type.hasPointers() && v.storage_class & STC.scope_ &&
+                if (exp.type.hasPointers() && v.storage_class & STC.scope_ &&
                     !(sc.flags & SCOPE.debug_) && sc.func.setUnsafe())
                 {
                     exp.error("cannot take address of `scope` %s `%s` in `@safe` function `%s`", p, v.toChars(), sc.func.toChars());
                     return false;
                 }
             }
-            else if (!(sc.flags & SCOPE.debug_) && sc.func.setUnsafe())
+            else if (!(sc.flags & SCOPE.debug_) &&
+                     sc.func.setUnsafe())
```

Comment:
Unrelated change ?

Reply:
No. It's reformatting.

Reply:
because 20691 will insert a line between them.

Reply:
It's an unrelated change. It doesn't have merit on its own (while the rest does).

Those kinds of change are hurtful to readability, because when you go back to it months later, you have no idea why such and such was done, as the context is lost. That's why I always ask, please, have changes that *stands on their own* and not within a context.

I'm going to auto-merge anyway, because the best is the enemy of the good, but please keep this in mind when making commit, just as we keep in mind to make single-commit PRs.